From: Jan Höppner <hoeppner@linux.vnet.ibm.com>
Subject: dasd: add kmsg descriptions for new ioctl BIODASDCHECKFMT
Patch-mainline: Never, kmsg in sles only
References: bsc#1025482, FATE#321527, LTC#146542

Summary:     dasd: Add new ioctl BIODASDCHECKFMT
Description: Implement new DASD IOCTL BIODASDCHECKFMT to check a range of
             tracks on a DASD volume for correct formatting. The following
             characteristics are checked:
             - Block size
             - ECKD key length
             - ECKD record ID
             - Number of records per track

Upstream-Description:

             dasd: add kmsg descriptions for new ioctl BIODASDCHECKFMT

Signed-off-by: Jan Höppner <hoeppner@linux.vnet.ibm.com>
Acked-by: John Jolly <jjolly@suse.com>
---
 Documentation/kmsg/s390/dasd |   12 ++++++++++++
 1 file changed, 12 insertions(+)

--- a/Documentation/kmsg/s390/dasd
+++ b/Documentation/kmsg/s390/dasd
@@ -49,6 +49,18 @@
  */
 
 /*?
+ * Text: "%s: The specified DASD is a partition and cannot be checked\n"
+ * Severity: Warning
+ * Parameter:
+ *   @1: bus ID of the DASD
+ * Description:
+ * The DASD you try to check is a partition. Partitions cannot be checked
+ * separately. You can only check a complete DASD including all its partitions.
+ * User action:
+ * Check the complete DASD.
+ */
+
+/*?
  * Text: "%s: Formatting unit %d failed with rc=%d\n"
  * Severity: Error
  * Parameter:
